test/integration/testdata/golden.v2.json

```diff
@@ -670,6 +670,26 @@
        "s": "foo"
       },
       "$ref": "#/definitions/defaults.SubStruct"
+     },
+     "defaultedAliasSymbolReference": {
+      "type": "string",
+      "default": "SymbolConstant"
+     },
+     "externalSymbolReference": {
+      "type": "string",
+      "default": "apple"
+     },
+     "fullyQualifiedSymbolReference": {
+      "type": "string",
+      "default": "SymbolConstant"
+     },
+     "localSymbolReference": {
+      "type": "string",
+      "default": "SymbolConstant"
+     },
+     "pointerConversionSymbolReference": {
+      "type": "string",
+      "default": "apple"
```

Comment:
Very cool to have the end-to-end test show that we end-up with the properly resolved values injected in the openapi! (big belated thanks to @seans3 for having written the test that makes the end-to-end possible).